engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java

```diff
@@ -94,4 +121,33 @@ public void emitTuples()
     private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
   }
 
+  private static class CommitAwareInputOperator extends CommitAwareOperator
+  {
+    public final transient DefaultOutputPort<String> output = new DefaultOutputPort<String>();
+  }
+
+  private static class CommitAwareProcessOperator extends CommitAwareOperator
+  {
+    private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
+    public static ArrayList<Long> committedWindowIds;
+    public static final synchronized List<Long> getCommittedWindowIdsContainer()
```

Comment:
Had taken [CommitAwareOperator](https://github.com/apache/apex-core/blob/master/engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java#L65) as a reference. I changed the fields from static to non-static variables/methods but the tests failed. This is because in StreamingContainer.deployNodes-> retrieveNode() we retrieve the node from InputStream and a new object is created. The committedWindowIds sink is populated in the newer object and the original object in the test-case has committedWindowIds empty which leads to the Assert exception.